packages/db-objects-file/lib/objects/objectsInMemServerRedis.js

```diff
@@ -698,17 +674,17 @@ class ObjectsInMemoryServer extends ObjectsInMemoryFileDB {
     /**
      * Destructor of the class. Called by shutting down.
      */
-    destroy() {
+    destroy(callback) {
```

Comment:
Wouldn't this be a good time to use Promises instead? Then you can await this method.

```js
return new Promise(resolve => {
    this.server.close(() => resolve());
});
```

Reply:
done